app/controllers/api/subcollections/metrics.rb

```diff
@@ -0,0 +1,17 @@
+module Api
+  module Subcollections
+    module Metrics
+      RESOURCE_TYPES = {
+        'vms' => 'VmOrTemplate'
```

Comment:
NOTE other collections have metrics also (e.g. hosts and container_groups), going to tackle that in a follow-up